proto/activity/v1alpha/subscriptions.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+package com.seed.activity.v1alpha;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+option go_package = "seed/backend/genproto/activity/v1alpha;activity";
+
+// Subscriptions service provides subscription capabilities.
+service Subscriptions {
+  // Subscribe to a document or space.
+  rpc Subscribe(SubscribeRequest) returns (google.protobuf.Empty);
+
+  // Remove a subscription.
+  rpc Unsubscribe(UnsubscribeRequest) returns (google.protobuf.Empty);
+
+  // Lists active subscriptions.
+  rpc ListSubscriptions(ListSubscriptionsRequest) returns (ListSubscriptionsResponse);
+}
+
+// Subscribe to a resource
+message SubscribeRequest {
```

Comment:
where is the path, @juligasa ?

Reply:
Oh, I maybe I should change `id` since it is ambiguous. Do you want an `url` field where you can input `hm://<account><path>` or you want two different fields `account`, `path` just like in `getDocument`? @ericvicenti

Reply:
In all the new APIs we use account and path separately.

Reply:
yeah i would like to keep the APIs consistent